pkgs/development/python-modules/devtools/default.nix

```diff
@@ -0,0 +1,27 @@
+{ buildPythonPackage, pythonOlder, fetchFromGitHub, lib, pygments
+, pytestCheckHook, pytest-mock }:
+buildPythonPackage rec {
+  pname = "devtools";
+  version = "0.6.1";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "samuelcolvin";
+    repo = "python-${pname}";
+    rev = "v${version}";
+    sha256 = "0s1d2jwijini7y1a3318yhb98mh1mw4pzlfx2zck3a8nqw984ki3";
+  };
+
+  propagatedBuildInputs = [ pygments ];
+
+  doCheck = true;
+  checkInputs = [ pytestCheckHook pytest-mock ];
+
+  meta = with lib; {
+    description =
+      "Python's missing debug print command and other development tools";
```

Comment:
```suggestion
    description = "Python's missing debug print command and other development tools";
```

Reply:
Thanks, I will make that change. I used `nixfmt` to format these. Is there a specific width setting that I should use in the future?

Reply:
> Is there a specific width setting that I should use in the future?

We don't really have one. Just keep it reasonable.